src/main/java/nl/novi/TechItEasy/controllers/WallBracketController.java

```diff
@@ -11,7 +11,7 @@
 import java.util.List;
 
 @RestController
-@RequestMapping("/api/WallBracket")
+@RequestMapping("/api/wallbracket")
 public class WallBracketController {
 
     private final WallBracketService wallBracketService;
```

Comment:
Pas op regel 35 eventueel op met gebruiken van afkortingen zoals de letter T die je daar gebruikt. Als je daar wallBracket neerzet bijvoorbeeld is het makkelijker onthouden wat die variabele echt is. Maar je hebt vast een beter geheugen dan ik als je kan onthouden waar die t voor staat :)